float64 directionDegrees
float64 speedKmph
